tests/hikari/test_guilds.py

```diff
@@ -239,6 +239,11 @@ def test_format_avatar(self, model, mock_user):
         mock_user.format_avatar.assert_called_once_with(ext="png", size=4096)
         assert result is mock_user.format_avatar.return_value
 
+    async def test_ban(self, model, mock_user):
+        result = await mock_user.ban(reason="cuz u was meen")
```

Comment:
to be more thorougher we'd probably want to test it with all the optional arguments being passed in 1 test and none of them being passed in another but this isn't too important

Reply:
@FasterSpeeding  I tested it with a test bot and it's working fine with or without reason